nncf/tensorflow/quantization/quantize_model.py

```diff
@@ -124,16 +124,7 @@ def _create_nncf_config(
             compression_config["ignored_scopes"] = _ignored_scope
 
     if advanced_parameters is not None:
-        advanced_config = convert_advanced_parameters_to_dict(advanced_parameters)
-
-        ranges = advanced_config.get("initializer", {}).get("range")
-        if ranges is not None:
-            for rconfig in ranges:
-                rconfig["num_init_samples"] = subset_size
-                if "type" not in rconfig:
-                    rconfig["type"] = DEFAULT_RANGE_TYPE
```

Comment:
it looks like you should set default parameters for "type" and "num_init_samples" in activation and weights ranges

Reply:
The `type` field set in `_get_default_quantization_config`  https://github.com/openvinotoolkit/nncf/blob/67a0be8b3b3848a6ba3bf23ee15f0df26e83050e/nncf/torch/quantization/quantize_model.py#L125

On this line types set again because `config.update` overrites `config["initializer"]`. I describe it in description of this PR.

Reply:
Fill `type` and `num_init_samples` in `apply_advanced_parameters_to_config` from part of config that generated by `_get_default_quantization_config`: https://github.com/openvinotoolkit/nncf/pull/1906/files#diff-950f24ab49d1eb3db259be96703abb72ea21125416cdf9b110357754a3bfaa8fR333-R336 

And add test to test this case.